bundles/org.openhab.binding.draytonwiser/src/main/resources/ESH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,415 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="draytonwiser"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+
+	<!-- Bridge Thing Type -->
+	<bridge-type id="heathub">
+		<label>HeatHub</label>
+		<description>A Drayton Wiser HeatHub acting as a bridge to Thermostats and TRVs</description>
+		<representation-property>serialNumber</representation-property>
+
+		<config-description>
+			<parameter name="networkAddress" type="text" required="true">
+				<context>network-address</context>
+				<label>Network Address</label>
+				<description>Network address of the HeatHub</description>
+			</parameter>
+			<parameter name="secret" type="text" required="true">
+				<label>Authorisation Token</label>
+				<description>Auth token required to access the API</description>
```

Comment:
password context maybe?

Reply:
It's not really a password though, nor an account specific access token. It's a hardcoded value built into the hub.

Reply:
leave as is then.